lib/net.js

```diff
@@ -1252,19 +1246,19 @@ function createServerHandle(address, port, addressType, fd) {
 }
 exports._createServerHandle = createServerHandle;
 
-
-Server.prototype._listen2 = function(address, port, addressType, backlog, fd) {
-  debug('listen2', address, port, addressType, backlog, fd);
+function setUpListenHandle(address, port, addressType, backlog, fd) {
```

Comment:
It may just be me but `setUp` looks odd. Perhaps `setupListenHandle` would be better?